features-json/urlsearchparams.json

```diff
@@ -0,0 +1,268 @@
+{
+  "title":"URLSearchParams",
+  "description":"The URLSearchParams interface defines utility methods to work with the query string of a URL.",
+  "spec":"https://url.spec.whatwg.org/#urlsearchparams",
+  "status":"ls",
+  "links":[
+    {
```

Comment:
Consider adding a link to MDN's reference docs:
https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams